addons/packages/ako-operator/1.4.0/README.md

```diff
@@ -42,3 +134,117 @@ The following configuration values can be set to customize the ako-operator inst
 ## Usage Example
 
 This walkthrough guides you through using ako-operator...
+
+### Verify your configurations
+
+After the package is installed and reconcile successfully, two AKODeploymentConfigs are deployed globally.  
+You can list and inspect them with `kubectl get akodeploymentconfig`. They are the configuration interfaces for the ako-operator.
+In general,
+
+* `install-ako-for-management-cluster` specifies the parameters to deploy AKO statefulset in the management cluster, which
+  then provides L4/L7 load balancing for applications.
+* `install-ako-for-all` specifies the parameters to deploy AKO statefulsets and reconcile users in each future workload clusters. ako-operator passes these parameters to each workload cluster via addon secrets to
+  ako-operator will watch AKODeploymentConfig CR objects, then create, and update the desired AKO deployment defined in akodeploymentconfig to each workload cluster.
+
+In the management cluster, check if AKO statefulset has been deployed successfully and also check AKO's log to make sure no persistent error messages.
+
+```shell
+# retrieve the pod name of the ako-operator manager
+kubectl get pods -n tkg-system-networking
+
+# check if the log is error-free.
+kubectl logs ako-operator-controller-manager-xxxxxxxx-xxxxx manager -n tkg-system-networking
+```
+
+### Per cluster configuration via akodeploymentconfig
+
+You can also create your own akodeploymentconfig for controlling cluster match a selector
+* First create a new akodeploymentconfig using the following template.
+* Under spec.clusterSelesector.matchLaebls put down the label that matches the cluster you want to the config to
+* AKO will monitor the label and apply the corresponding config to the matched clusters.
+```yaml
+apiVersion: networking.tkg.tanzu.vmware.com/v1alpha1
+kind: AKODeploymentConfig
+metadata:
+    name: install-ako-for-management-cluster
```

Comment:
please change the name here to something other than `install-ako-for-management-cluster`, we don't want to confuse users.